src/Elastic.CommonSchema/Serialization/BaseJsonFormatter.Generated.cs

```diff
@@ -86,13 +87,13 @@ static bool ReadString(ref JsonReader reader, ref string stringProp)
 				stringProp = reader.ReadString();
 				return true;
 			}
-			
+
 			while (reader.ReadIsInObject(ref count))
 			{
 				var propertyName = reader.ReadPropertyNameSegmentRaw();
 				if (AutomataDictionary.TryGetValue(propertyName, out var value))
 				{
-					_ = value switch
+					var read = _ = value switch
```

Comment:
Dont forget to update the .cshtml template and regen.

Reply:
Updated as part of the move to `System.Text.Json` https://github.com/elastic/ecs-dotnet/pull/9